src/contract/deploy.rs

```diff
@@ -49,6 +49,61 @@ impl<T: Transport> Builder<T> {
     where
         P: Tokenize,
         V: AsRef<str>,
+    {
+        let transport = self.eth.transport().clone();
+        let poll_interval = self.poll_interval;
+        let confirmations = self.confirmations;
+
+        self.do_execute(code, params, from, move |tx| {
+            confirm::send_transaction_with_confirmation(transport, tx, poll_interval, confirmations)
+        })
+    }
+    /// Execute deployment passing code and contructor parameters.
+    ///
+    /// Unlike the above `execute`, this method uses
+    /// `sign_raw_transaction_with_confirmation` instead of
+    /// `sign_transaction_with_confirmation`, which requires the account from
+    /// which the transaction is sent to be unlocked.
+    pub fn sign_and_execute<P, V, Ft>(
```

Comment:
The `Ft` parameter is unused. This is problematic for calling code because the compiler cannot infer the instantiation of `Ft`. This leads to the calling code always requiring an explicit type annotation.

Reply:
Let's just remove it then. I guess I left it by mistake.

Reply:
I'll try this out.

Reply:
Does it solve the issues you were having?

Reply:
It did. I get other errors now, but as far as this PR goes, it has what we need.
Do you still want me to add a test before we merge?